template-contributing-guide.md

```diff
@@ -0,0 +1,344 @@
+    <td>Optional</td>
+  </tr>
+  <tr>
+    <td>Template example</td>
+    <td>The Good Docs Project is in the process of designing a fake documentation project that can provide examples of our templates in action. Until this project is ready, it is optional to provide an example of the template. However, if you want to create a fake example as part of your template set, you may do so.</td>
+    <td>Optional</td>
+  </tr>
+</table>
+
+For more detailed information about each deliverable, see [Template Deliverables](template-deliverables.md).
+
+
+## Overview of the template writing phases
+Contributing a template project to our repository has five phases, as summarized in this table:
+
+<table>
+  <tr>
+    <th>Phase</th>
+    <th>Your goals</th>
+    <th>Resources to help you</th>
+  </tr>
+  <tr>
+    <td>1 - Join the community</td>
+    <td><ul>
+          <li>Join our project’s communication channels.</li>
+          <li>Decide which initiative you’d like to work on based on your interests and experience.</li>
+        </ul>
+    </td>
+    <td><ul>
+          <li>The Good Docs Project welcoming committee</li>
+          <li>Working groups</li>
+        </ul>
+    </td>
+  </tr>
+  <tr>
+    <td>2 - Adopt a template</td>
+    <td><ul>
+          <li>Join a template writing working group led by a template mentor.</li>
+          <li>Work with a template mentor and your working group to decide which template you’ll work on.</li>
+          <li>Assign yourself to the corresponding issue for that template.</li>
+        </ul>
+    </td>
+    <td><ul>
+          <li>The issues board on the templates repo</li>
+          <li>Template mentors</li>
+          <li>Templates coordinator</li>
+          <li>Template working groups</li>
+        </ul>
+    </td>
+  </tr>
+  <tr>
+    <td>3 - Research and draft</td>
+    <td><ul>
+          <li>Research examples and best practices for the type of template you are working on.</li>
+          <li>Collaborate and get early feedback on your research from your template mentor and/or other templateers as part of a template writing working group.</li>
+          <li>Create drafts of the template set deliverables in Google Docs or your preferred tool.</li>
+        </ul>
+    </td>
+    <td><ul>
+          <li>Working groups</li>
+          <li>Template mentors</li>
+          <li>Templates coordinator</li>
+        </ul>
+    </td>
+  </tr>
+  <tr>
+    <td>4 - Get feedback on drafts from the community</td>
+    <td><ul>
+          <li>Revise and refine the drafts of your deliverables based on feedback from at least three community reviewers. Community reviewers are anyone who is a member of the Good Docs Project, including members of template writing working groups.</li>

```

Comment:
Suggestion: non-blocking: My recollection of our last meeting was that we have a "strong preference for three or more reviewers" and not make it a hard requirement.
```suggestion
          <li>Revise and refine the drafts of your deliverables. Expect to receive feedback from at least three community reviewers. Community reviewers are anyone who is a member of the Good Docs Project, including members of template writing working groups.</li>

```